settings.d/dynflow.yml.example

```diff
@@ -1,3 +1,5 @@
 ---
 :enabled: true
 :database: /var/lib/foreman-proxy/dynflow/dynflow.sqlite
+:console_authorize_with_ssl_client: true
+:console_authorize_with_trusted_hosts: false
```

Comment:
This gets resolved to something like `--foreman-proxy-dynflow-console-trusted-hosts-auth`  or similar, it's still very long.

What about just making this one setting? Say, `console_auth` which would turn off both trusted_hosts and ssl.   I would guess the only reason to do this is in a situation where we want to collect some debug information and have the user directly access the console. Right?

In every other situation, I think we should let the smart proxy handle it according to its own settings.